### PR TITLE
Hts 132 d

### PR DESCRIPTION
**Pull request by Jackie1988**

### CODE REVIEW

File: project/FrontendBuild.scala [hunk 1]
```diff
@@ -11,6 +11,7 @@ object FrontendBuild extends Build with MicroService {
   val compile = Seq(
     ws,
     "uk.gov.hmrc" %% "frontend-bootstrap" % "8.7.0",
+    "uk.gov.hmrc" %% "play-partials" % "6.1.0",
```
Comment (AndyHWChung) [inline project/FrontendBuild.scala]: do we need this anymore?

Comment (Jackie1988) [reply project/FrontendBuild.scala]: Nope, its gone!

File: conf/application.conf [hunk 2]
```diff
@@ -362,3 +363,6 @@ health {
 # performed. The IP address is looked for in the HTTP header of requests
 # 'true client IP'
 http-header-ip-whitelist = [ ]
+
+# to learn why this was included: https://confluence.tools.tax.service.gov.uk/display/TEC/2016/03/14/Setting+Security+Headers+in+frontend+services
+play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9032 localhost:9250 www.google-analytics.com data:"
```
Comment (AndyHWChung) [inline conf/application.conf]: i think we already had this setting in `application.conf` - this looks like a duplicate

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**